include/pmacc/cuSTL/algorithm/kernel/detail/SphericMapper.hpp

```diff
@@ -53,7 +53,7 @@ namespace pmacc
                 {
                     static constexpr int dim = 1;
 
-                    typename math::Size_t<3>::BaseType cuplaGridDim(const math::Size_t<1>& size) const
+                    math::Size_t<3> cuplaGridDim(const math::Size_t<1>& size) const
```

Comment:
Can't this return type and similarly in other methods be `auto`?

Reply:
Yes it can. I think clang-tidy can do such a transformation. I will run it again after this PR.